### DISCUSSION TITLE
can we subscribe to view collection in sveltekit?

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#4019] [Q&A] [answered]
if not, my problem is i want ot subscribe to 3 collections at a time, i thought of joining those collection into a view and subscribe that, if  subscribing to view not possible is there any better way to subscribe to 3 collections at a time

**Comment by ganigeorgiev** [comment, marked as answer]
No, views don't emit realtime events because they don't have create/update/delete actions.

> ... my problem is i want ot subscribe to 3 collections at a time ...

I'm not sure that I understand your use case but you can call `subscribe()` for each individual collection. There shouldn't be any noticeable overhead because the SDKs maintains only a single long-lived connection per instance/tab/window even if you have multiple subscriptions.
